tests/unit/transpiler/test_execute.py

```diff
@@ -33,7 +33,8 @@ def safe_remove_file(file_path: Path):
 
 def write_data_to_file(path: Path, content: str):
     with path.open("w") as writable:
-        writable.write(content)
+        # writable.write(content)
+        writable.write(content.encode("utf-8", "ignore").decode("utf-8"))
```

Comment:
why is this necessary?

Reply:
Can you add a comment?

Reply:
I have added this because I was not able to write the query - `1SELECT ~v\ud83d' ` for `TOKEN ERROR` scenario

Reply:
just add that comment please and remove line 36

Reply:
done